backend/routes/webAuthn.go

```diff
@@ -13,5 +13,6 @@ func WebAuthnRoute() chi.Router {
 	r.Post("/authorize/request", controller.AuthorizeRequest)
 	r.Post("/authorize/response/{doorId}/{challenge}", controller.AuthorizeResponse)
 	r.Get("/authorize/{doorId}/{yubiKeyId}", controller.GetYubiKeyDoorAccess)
+	r.Get("/unlocked/{doorId}", controller.IsDoorOpen)
```

Comment:
Did we make the necessary embedded changes to account for this?

Reply:
Yes.